### DISCUSSION TITLE
Will rdio-scanner run on a Dell x720xd with a Linux OS?

### DISCUSSION THREAD

**Discussion by kd4e** [chuot/rdio-scanner#260] [Q&A] [answered]
The Dell x720xd uses a pair of x86 compatible Xeon ES-2697 v2 processors.
I have TrueNas Scale (Linux) running on it.
Is there a reasonable possibility that I could host rdio-scanner on it?
I have trunk-recorder running on a RPi as the source.
I'd like to build a robust gui where users may log in and make a lots of personalized listening choices.
I'm hoping to avoid dealing with docker ...
Thanks

**Comment by chuot** [comment, marked as answer]
I do use TrueNAS so I don't know much about it, but I installed it quickly to see how it looks. Well, I think your best choice is indeed to use the Docker version of Rdio Scanner which is available [here](https://hub.docker.com/repository/docker/chuot/rdio-scanner). There may be another way of doing it, but again I don't know much about TrueNAS.
